lms/djangoapps/courseware/access_response.py

```diff
@@ -0,0 +1,85 @@
+"""
+This file contains all the classes used by has_access for error handling
+"""
+
+
+class AccessResponse(object):
+    """Class that represents a response from a has_access permission check."""
+    def __init__(self, has_access, error_code=None, developer_message=None, user_message=None):
+        """
+        :param has_access (bool): if the user is granted access or not
```

Comment:
Check the edX coding conventions for specifying function parameters.  We don't use the `:param` style.